core/sbt-cloudflow/src/main/scala/cloudflow/sbt/CommonSettingsAndTasksPlugin.scala

```diff
@@ -42,7 +42,7 @@ object CommonSettingsAndTasksPlugin extends AutoPlugin {
 
   // common definitions
   final val CloudflowLocalConfigFile = ".lightbend/cloudflow/pipectl.json"
-  final val CloudflowDockerBaseImage = "lightbend/cloudflow-base:1.3.1-spark-2.4.5-flink-1.10.0-scala-2.12"
+  final val CloudflowDockerBaseImage = "lightbend/cloudflow-base:1.3.2-spark-2.4.5-flink-1.10.0-scala-2.12"
```

Comment:
Have we pushed this image?

Reply:
not yet ..

Reply:
done ..

Reply:
@debasishg IMO it should be 1.4.0, since that is the "multi-image" release and multi-image will never be released for 1.3.x

Reply:
ah .. ok .. will change ..

Reply:
I did not change this to `1.4.0` since this is the old base image which will be released with `1.3.4` .. But I changed the base images of the multi-image plugins to `1.4.0` ..